edad.rb

```diff
@@ -0,0 +1,13 @@
+require 'date'
```

Comment:
It was super good, it is not an error but as a general rule we always try to do everything in English in ruby, names of variables, methods, models, classes at least it is the standard we have here, so as not to combine English and Spanish as it usually happens, for what the age class should go in English, so that you keep it in mind.